packages/calcite-tokens/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@esri/calcite-design-tokens",
+  "version": "1.0.0-beta.0",
+  "description": "Esri's Calcite Design System Tokens",
+  "homepage": "https://github.com/Esri/calcite-design-tokens",
+  "keywords": [
+    "Calcite",
+    "Calcite Design System",
+    "design tokens",
+    "tokens"
+  ],
+  "main": "./dist/js/calcite-headless.js",
+  "type": "module",
```

Comment:
Nitpick: I prefer explicitly specifying the files to include in release tarballs using the `files` property in `package.json` instead of excluding via an npmignore file. Plus npm uses gitignore if there is no npmignore.

ref: https://docs.npmjs.com/cli/v6/configuring-npm/package-json#files